src/main/java/frc/team3388/robot/RobotMap.java

```diff
@@ -14,4 +14,9 @@
     public static final int PISTON_REVERSE_CHANNEL = 2;
     public static final int PISTON_FORWARD_CHANNEL = 3;
 
+    // drive motors
+    public static final int RIGHT1 = 1;
```

Comment:
Add `DRIVE_` prefix to all of the constants here. It will help recognize what this means.

Reply:
fixed